README.md

```diff
@@ -1,161 +1,157 @@
-## Google Cloud Storage: Node.js Client
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-[![CircleCI](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-node-storage.svg?style=shield)](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-node-storage)
-[![codecov](https://codecov.io/gh/GoogleCloudPlatform/google-cloud-node-storage/branch/repo-migration/graph/badge.svg)](https://codecov.io/gh/GoogleCloudPlatform/google-cloud-node-storage)
+# Google Cloud Storage: Node.js Client
 
-Google Cloud Storage allows world-wide storage and retrieval of any amount
-of data at any time. You can use Google Cloud Storage for a range of scenarios,
-including serving website content, storing data for archival and disaster
-recovery, or distributing large data objects to users via direct download.
+[![Release quality](https://img.shields.io/badge/Release%20quality-General%20Availability%20(GA)-brightgreen.svg?style&#x3D;flat)](https://cloud.google.com/terms/launch-stages)
+[![CircleCI](https://img.shields.io/circleci/project/github/GoogleCloudPlatform/google-cloud-node-storage.svg?style=flat)](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-node-storage)
+[![codecov](https://img.shields.io/codecov/c/github/GoogleCloudPlatform/google-cloud-node-storage/repo-migration.svg?style=flat)](https://codecov.io/gh/GoogleCloudPlatform/google-cloud-node-storage)
 
-- [Node.js API Reference][node-storage-docs]
-- [Google Cloud Storage Documentation][cloud-storage-docs]
+> Node.js idiomatic client for [Cloud Storage][product-docs].
 
+[Cloud Storage](https://cloud.google.com/storage/docs) allows world-wide storage and retrieval of any amount of data at any time. You can use Google Cloud Storage for a range of scenarios including serving website content, storing data for archival and disaster recovery, or distributing large data objects to users via direct download.
 
-## Installation
+* [Cloud Storage Node.js Client API Reference][client-docs]
+* [Cloud Storage Documentation][product-docs]
 
-```sh
-$ npm install --save @google-cloud/storage
-```
+Read more about the client libraries for Cloud APIs, including the older
+Google APIs Client Libraries, in [Client Libraries Explained][explained].
 
-## Quick Start
+[explained]: https://cloud.google.com/apis/docs/client-libraries-explained
 
-### Instantiate a client
+**Table of contents:**
 
-```js
-var gcs = require('@google-cloud/storage')({
-  projectId: 'grape-spaceship-123',
-  keyFilename: '/path/to/keyfile.json'
-});
-```
+* [QuickStart](#quickstart)
```

Comment:
Note, this was modeled after: https://cloud.google.com/natural-language/docs/quickstart-client-libraries